src/main/java/eu/dissco/backend/repository/MasJobRecordRepository.java

```diff
@@ -101,6 +102,17 @@ public void markMasJobRecordsAsFailed(List<UUID> ids) {
         .execute();
   }
 
+  public void markMasJobRecordAsRunning(String creatorId, UUID masJobId) throws NotFoundException {
+    var updated = context.update(MAS_JOB_RECORD)
+        .set(MAS_JOB_RECORD.STATE, AnnotationState.RUNNING.getState())
```

Comment:
At some point it would be nice to have these in db enums as well. But not real prio, just saves space and make sure it fails if something else goes into that column https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
made a jira story